manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: BinaryNinja.Demo
+PackageVersion: idk
+PackageLocale: en-US
+Publisher: Binary Ninja

```

```suggestion
Publisher: Vectore 35, Inc.
```
Must match control panel